server/api/impl/proxy/manifest.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
```

Comment:
These are convenience methods.

As far as proxy is concerned, manifest is just text, and comes (usually) from the Marketplace, with the `InstallApp` API call.